docs/advisories/cve_2017_14491.md

```diff
@@ -0,0 +1,41 @@
+# CVE-2017-14491
+
+A vulnerability in dnsmasq, used by kube-dns, requires an upgrade to the kube-dns component.
+
+[More details](https://security.googleblog.com/2017/10/behind-masq-yet-more-dns-and-dhcp.html)
+
+## Fixed in kops versions:
+
+Not yet released.  Fixed by @mikesplain in [#3511](https://github.com/kubernetes/kops/pull/3511)
+
+## Hotfix for Kubernetes
+
+The minimal fix is to just update the image for the dnsmasq pod:
+
+```
+kubectl set image deployment/kube-dns -n kube-system dnsmasq=gcr.io/google_containers/k8s-dns-dnsmasq-amd64:1.14.5
```

Comment:
Should the image name here be `gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.5`?